src/frontend/src/navigation/RouterConfig.tsx

```diff
@@ -0,0 +1,32 @@
+import { Dashboard, Project, Group, Layout } from 'pages';
+import type { RouteObject } from 'react-router';
+import AppRoutes from 'navigation/AppRoutes';
+
+const routes: RouteObject[] = [
```

Comment:
moved this out to navigation,

also pulled the type from the package directly instead of redefining it